Makefile

```diff
@@ -165,146 +165,81 @@ protoc:
 # mockgen must be installed on the system to make this work.
 # Install it by running:
 # `go get github.com/golang/mock/mockgen`.
-mockgen = $(GOPATH)/bin/mockgen
+mockgen = GO111MODULES=on $(GOPATH)/bin/mockgen
+
+define kraken_mockgen
```

Comment:
Use `mock$(notdir $(1))` to derive the package name instead. This will make the calls slimmer.

Reply:
Done.